composer.json

```diff
@@ -1,25 +1,26 @@
 {
-	"name": "wedy/php-resque-pause",
-	"type": "library",
-	"description": "php-resque-pause is a PHP port of resque-pause, which adds support for pausing / unpausing jobs to Resque.",
-	"authors": [
-		{
-			"name": "Wedy Chainy",
-			"email": "askwedi@wedipedia.org"
-		}
-	],
-	"repositories": [
-		{ "type": "git", "url": "https://github.com/chrisboulton/php-resque" }
-	],
-	"require": {
-		"chrisboulton/php-resque": "dev-master"
-	},
+  "name": "wedy/php-resque-pause",
+  "type": "library",
+  "description": "php-resque-pause is a PHP port of resque-pause, which adds support for pausing / unpausing Resque jobs.",
+  "authors": [{
+    "name": "Wedy Chainy",
+    "email": "askwedi@wedipedia.org"
+  }],
+  "repositories": [{
+    "type": "git",
+    "url": "https://github.com/chrisboulton/php-resque"
+  }],
+  "require": {
+    "chrisboulton/php-resque": "dev-master"
+  },
   "require-dev": {
-    "phpunit/phpunit": "3.7.*"
+    "phpunit/phpunit": "3.7.*",
```

Comment:
Unless you have a particular reason to lock to this old version of phpunit, the latest stable version is 4.5.